plugins/redis/secret_key.go

```diff
@@ -0,0 +1,55 @@
+package redis
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func SecretKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.SecretKey,
+		DocsURL: sdk.URL("https://docs.redis.com/latest/rc/api/get-started/manage-api-keys/"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.AccessKey,
```

Comment:
Does Redis call these Access Keys anywhere?

Reply:
This is the credential for redis cloud. The official redis cloud terraform provider uses a secret access key: https://github.com/RedisLabs/terraform-provider-rediscloud

Reply:
Aha. Looking a bit closer at Redis' documentation, I think "Access Key" and "Secret Key" are the legacy names:

![image](https://github.com/1Password/shell-plugins/assets/1965218/76bff2a8-c171-4dfc-8a1e-5f568e9520ee)

I think it'd be best to stick to the most up-to-date names.